lor-init

```diff
@@ -40,7 +40,7 @@ function lor-config() {
 	local stack_name="$(basename "$(dirname "$(fgrep 'stack_prefix:' "${LOR_DIR}/group_vars/"*/vars.yml | fgrep "${stack_prefix}" | sed 's|:.*||')")")"
```

Comment:
```suggestion
	local stack_name="$(basename "$(dirname "$(fgrep 'stack_prefix:' "${LOR_DIR}/group_vars/"*/vars.yml | fgrep -w "${stack_prefix}" | sed 's|:.*||')")")"
```
Just hit the issue. This command will return last value even if the user writes by accident "n" instead of "nb" (an in my case it configured "wingedhelix"). The "-w" makes sure, that the stack prefix matches completely, not accidentally.